match2/wikis/starcraft2/match_group_legacy_default.lua

```diff
@@ -153,5 +153,56 @@ function p.addMaps(match)
 	return match
 end
 
-return p
+--this is for custom mappings
```

Comment:
I think this comment would benefit from defining "custom mappings" a little more, it's not an immediately obvious term.

Reply:
something like the following?
```--[[
custom mappings are used to overwrite the default mappings
in the cases where the default mappings do not fit the
parameter format of the old bracket
]]--

Reply:
Yep!

Reply:
added it a bit further up because we reference custom mappings there already